content/docs/rfcs/16/README.md

```diff
@@ -81,6 +81,10 @@ The following structured types are defined for use on the Debug API:
 | `listenAddresses` | `Array`[`String`] | mandatory | Listening addresses of the node |
 | `enrUri` | `String` | optional | ENR URI of the node |
 
+#### WakuInfo
+
+`Version`
```

Comment:
Is that a typo? I am not sure to understand why this text is here.

Reply:
Yes :sweat_smile: . Removed in 1675b76b16